src/executionFiles/utils.ts

```diff
@@ -152,7 +163,7 @@ export const getTransferredNativeTokenBalanceFromLog = (params: {
     try {
       const parsed = interfaceGas.parseLog(log)
       const result = {
-        tokenAddress: log.address.toLowerCase(),
+        toTokenAddress: log.address.toLowerCase(),
```

Comment:
Please take extra care here, I think this was wrong before but I could be wrong.

Reply:
Oh yes, that should be toTokenAddress. This function is not used/needed a lot, that's why we may have missed it.